spec/01-unit/002-conf_loader_spec.lua

```diff
@@ -85,6 +85,12 @@ describe("Configuration loader", function()
     assert.equal(8443, conf.proxy_ssl_port)
   end)
   it("attaches prefix paths", function()
+    -- patch os.getenv to not have existing environment variables interfere
+    local _getenv = os.getenv
+    os.getenv = function() end  -- luacheck: ignore
+    finally(function()
+        os.getenv = _getenv     -- luacheck: ignore
+      end)
```

Comment:
What about all the other places where we are calling `conf_loader()` in this test suite?

Reply:
yes, I had an itch about those. I fixed the ones I ran into, but it would probably be better to check all, and then most likely do this in the `setup` or `before_each` handlers.